### PR TITLE
Implement LoadScreen as far as possible

### PR DESCRIPTION
**Pull request by duncanspumpkin**
Annoyingly barely any of this can be done due to missing `GameWindowManager`

### CODE REVIEW

File: src/game/client/gui/loadscreen.cpp [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+#include "gameengine.h"
+#ifdef GAME_DLL
+#include "hooker.h"
+#endif
+
+// zh: 0x00608620 wb: 0x009C3F31
+LoadScreen::~LoadScreen()
+{
+#ifdef GAME_DLL
+    Call_Method<void, LoadScreen>(PICK_ADDRESS(0x00608620, 0x009C3F31), this);
```
Comment (jonwil) [inline src/game/client/gui/loadscreen.cpp]: WB address seems to be 0x009C3F11 and not 0x009C3F31

**Review by jonwil — CHANGES_REQUESTED** [review]
Fix that one address and I will approve this.

File: src/game/client/gui/loadscreen.h [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+class GameWindow;
+class GameInfo;
+
+class LoadScreen
+{
+public:
+    virtual ~LoadScreen();
+    // wb: 0x009C3EF0
+    LoadScreen() : m_loadScreen(nullptr) {}
```
Comment (tomsons26) [inline src/game/client/gui/loadscreen.h:26]: ctor before dtor

**Review by jonwil — APPROVED** [review]
Looks good to me now, approved.

**Pull request merged**